apps/api/src/database/repositories/competition-repository.ts

```diff
@@ -840,7 +840,18 @@ async function get24hSnapshotsImpl(
   }
 
   try {
-    const twentyFourHoursAgo = new Date(Date.now() - 24 * 60 * 60 * 1000);
+    const comp = await dbRead
```

Comment:
This is needed to show 24 hour stats after a comp is ended.  We probably shouldn't be doing any of the calculations in this function per request but that's for another PR.